### ISSUE TITLE
The method using(HttpConsumer.Factory) is undefined for the type ServiceQuery

### ISSUE THREAD

**Issue by InRiPa** [arrowhead-f/arrowhead-kalix-examples#4]
Hi,

I tried building the echo-cloud example. I get the error that the `using` method is undefined. Is this a known issue? Can you tell me how I can fix it? 

**System:**
> Os: ArchLinux 5.9.6-arch1-1
> Java: 
> openjdk version "14.0.2" 2020-07-14
>          OpenJDK Runtime Environment (build 14.0.2+12)
>          OpenJDK 64-Bit Server VM (build 14.0.2+12, mixed mode)

**Steps:** 

1. clone repo
2. navigate to echo-cloud folder
3. `$> mvn clean install`

```
[INFO] Scanning for projects...
[INFO] 
[INFO] -------------------< se.arkalix.examples:echo-cloud >-------------------
[INFO] Building Echo System and Client (Cloud) 0.5.0
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] 
[INFO] --- maven-clean-plugin:2.5:clean (default-clean) @ echo-cloud ---
[INFO] 
[INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ echo-cloud ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] skip non existing resourceDirectory /home/user/workspace/arrowhead-kalix-examples/echo-cloud/src/main/resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.8.1:compile (default-compile) @ echo-cloud ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 3 source files to /home/user/workspace/arrowhead-kalix-examples/echo-cloud/target/classes
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /home/user/workspace/arrowhead-kalix-examples/echo-cloud/src/main/java/se/arkalix/examples/EchoConsumer.java:[81,17] cannot find symbol
  symbol:   method using(se.arkalix.net.http.consumer.HttpConsumer.Factory)
  location: class se.arkalix.query.ServiceQuery
[ERROR] /home/user/workspace/arrowhead-kalix-examples/echo-cloud/src/main/java/se/arkalix/examples/EchoConsumer.java:[99,17] cannot find symbol
  symbol:   method using(se.arkalix.net.http.consumer.HttpConsumer.Factory)
  location: class se.arkalix.query.ServiceQuery
[INFO] 2 errors 
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  1.406 s
[INFO] Finished at: 2020-11-11T11:06:44+01:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.1:compile (default-compile) on project echo-cloud: Compilation failure: Compilation failure: 
[ERROR] /home/user/workspace/arrowhead-kalix-examples/echo-cloud/src/main/java/se/arkalix/examples/EchoConsumer.java:[81,17] cannot find symbol
[ERROR]   symbol:   method using(se.arkalix.net.http.consumer.HttpConsumer.Factory)
[ERROR]   location: class se.arkalix.query.ServiceQuery
[ERROR] /home/user/workspace/arrowhead-kalix-examples/echo-cloud/src/main/java/se/arkalix/examples/EchoConsumer.java:[99,17] cannot find symbol
[ERROR]   symbol:   method using(se.arkalix.net.http.consumer.HttpConsumer.Factory)
[ERROR]   location: class se.arkalix.query.ServiceQuery
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
```

**Comment by emanuelpalm** [comment]
The method should be `oneUsing()` rather than `using()`. It seems like I missed changing the name when upgrading to Kalix 0.5.0. Thank you for reporting the issue.

**Comment by InRiPa** [comment]
Thanks for addressing the issue. There is still a `using()` reference. I created a pull request for it. With that being replaced, I was abl to compile the example.

**Comment by emanuelpalm** [comment]
Great! Thank you! :+1:

**Comment by emanuelpalm** [comment]
Merged #5. Reopen the issue if you believe there is still something about this bug that should be fixed.
